index.js

```diff
@@ -1,32 +1,40 @@
 ﻿/**
-  * @INFO
-  * Bot Coded by Tomato#6966 | https://github.com/Tomato6966/discord-js-lavalink-Music-Bot-erela-js
-  * @INFO
-  * Work for Milrato Development | https://milrato.eu
-  * @INFO
-  * Please mention Him / Milrato Development, when using this Code!
-  * @INFO
-*/
+ * @INFO
+ * Bot Coded by Tomato#6966 | https://github.com/Tomato6966/discord-js-lavalink-Music-Bot-erela-js
+ * @INFO
+ * Work for Milrato Development | https://milrato.eu
+ * @INFO
+ * Please mention Him / Milrato Development, when using this Code!
+ * @INFO
+ */
 //Importing all needed Commands
 const Discord = require("discord.js"); //this is the official discord.js wrapper for the Discord Api, which we use!
 const colors = require("colors"); //this Package is used, to change the colors of our Console! (optional and doesnt effect performance)
 const Enmap = require("enmap"); //this package is our Database! We will use it to save the data for ever!
 const fs = require("fs"); //this package is for reading files and getting their inputs
+const {SlashCreator, GatewayServer} = require('slash-create');
+const path = require('path');
 
 //Extending structures
 require("./structures.js");
 
 //Creating the Discord.js Client for This Bot with some default settings ;) and with partials, so you can fetch OLD messages
 const client = new Discord.Client({
-  intents: 32387,
+    intents: 32387,
 //  messageCacheLifetime: 60,
-  fetchAllMembers: true,
+    fetchAllMembers: true,
 //  messageCacheMaxSize: 10,
-  restTimeOffset: 0,
-  restWsBridgetimeout: 100,
-  disableEveryone: true,
-  partials: ['MESSAGE', 'CHANNEL', 'REACTION'],
+    restTimeOffset: 0,
+    restWsBridgetimeout: 100,
+    disableEveryone: true,
+    partials: ['MESSAGE', 'CHANNEL', 'REACTION'],
 });
+const creator = new SlashCreator({
+    applicationID: require("./botconfig/config.json").applicationID,
```

Comment:
Require once before creating the const creator to avoid requiring the same file three times